pyproject.toml

```diff
@@ -13,11 +13,11 @@ include = ["CHANGES.md"]
 [tool.poetry.dependencies]
 python = "^3.6"
 click = "7.0+"
-docparse = "*"
+docstring_parser = "^0.8.1"
```

Comment:
This actually doesn't work. But the version exists and can be pip installed https://pypi.org/project/docstring-parser/0.8.1/
I haven't used poetry before, so I'm not quite sure why it wouldn't pick it up. Ideas?

Reply:
It works for me:
![image](https://user-images.githubusercontent.com/125854/125668052-404b1116-9e29-4758-816e-3ca25819189e.png)

Maybe try doing it within a clean virtualenv?

Reply:
Turned out the lock file was in a broken state or something. `poetry install` matches your screenshot after deleting the lock and reinstalling.